facade/src/main/java/com/redhat/lightblue/migrator/facade/ServiceFacade.java

```diff
@@ -357,4 +397,19 @@ public TimeoutConfiguration getTimeoutConfiguration() {
     public void setTimeoutConfiguration(TimeoutConfiguration timeoutConfiguration) {
         this.timeoutConfiguration = timeoutConfiguration;
     }
+
+    /**
+     * Shutdown thread pool executor for all facades.
+     *
+     */
+    public static void shutdown() {
```

Comment:
But it isn't.
http://stackoverflow.com/questions/7602706/ejb-application-shutdown-hook

On Wed, May 18, 2016 at 9:15 AM, Marek <notifications@github.com> wrote:

> In
> facade/src/main/java/com/redhat/lightblue/migrator/facade/ServiceFacade.java
> <https://github.com/lightblue-platform/lightblue-migrator/pull/350#discussion_r63699573>
> :
>
> > @@ -357,4 +397,19 @@ public TimeoutConfiguration getTimeoutConfiguration() {
> >      public void setTimeoutConfiguration(TimeoutConfiguration timeoutConfiguration) {
> >          this.timeoutConfiguration = timeoutConfiguration;
> >      }
> > +
> > +    /**
> > +     * Shutdown thread pool executor for all facades.
> > +     *
> > +     */
> > +    public static void shutdown() {
>
> It's here for the sake of completeness. Could be called on service
> undeployment to avoid leaks during hot deployment.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-migrator/pull/350/files/17b676bee35728c0487e52415290b9e02d5b9cc0#r63699573>
>